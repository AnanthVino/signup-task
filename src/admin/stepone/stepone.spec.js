import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import StepOne from '.';
    
describe('Admin StepOne', () => {
  it('should render correctly', () => {
    
    const wrapper = shallow(<StepOne />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})