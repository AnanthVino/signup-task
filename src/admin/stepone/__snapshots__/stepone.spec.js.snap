// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Admin StepOne should render correctly 1`] = `
<div
  className="container d-flex flex-column justify-content"
>
  <form>
    <div
      className="form-group"
    >
      <label>
        Firstname
      </label>
      <input
        className="form-control input-box rounded-0"
        defaultValue={null}
        onChange={[Function]}
        placeholder="First Name"
        type="text"
      />
    </div>
    <div
      className="form-group"
    >
      <label>
        Lastname
      </label>
      <input
        className="form-control input-box rounded-0"
        defaultValue={null}
        onChange={[Function]}
        placeholder="Last Name"
        type="text"
      />
    </div>
    <div
      className="form-group"
    >
      <label>
        Email
      </label>
      <input
        className="form-control input-box rounded-0"
        defaultValue={null}
        onChange={[Function]}
        placeholder="Email"
        required={true}
        type="text"
      />
    </div>
  </form>
  <div
    className="d-flex justify-content-between"
  >
    <a
      className="btn btn-primary rounded-0"
      href="/"
    >
      Go Back
    </a>
    <button
      className="btn btn-primary rounded-0"
      onClick={[Function]}
    >
      Next Step
    </button>
  </div>
</div>
`;
